.circleci/config.yml

```diff
@@ -1,12 +1,35 @@
 defaults: &defaults
   docker:
     - image: 087285199408.dkr.ecr.us-east-1.amazonaws.com/circle-ci-test-image-base:go1.16-go111module
```

Comment:
Ohhhhh this is using the docker container! Ah I totally misled you! In this case, the fix is to bump the tag here to the latest: `go1.16-tf1.0-tg31.1-pck1.7` (got that from AWS console in `phxdevops`)

Reply:
Ah no worries! I was wondering why the format of this config.yml was so different :) 

Updated in 3fa1006